packages/lib/client-shared/src/util.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * Settings for calling {@link gpcArtifactDownloadURL} as parsed from an
+ * environment variable by {@link parseGPCArtifactsConfig}.
+ */
+export type GPCArtifactsConfigEnv = {
+  source: string | undefined;
+  stability: string | undefined;
+  version: string | undefined;
+};
+
+/**
+ * The version of the latest supported version of artifacts published on NPM.
+ */
+export const GPC_ARTIFACTS_NPM_VERSION = "0.0.2";
```

Comment:
im not sure hard-coding this here makes the most sense as a long term solution but probably ok for now. later, it may be good to pass it in from `passport-client` (/`consumer-client`), where this value could be passed in via an environment variable.

Reply:
Yeah, this isn't a great place for it.  It really wants to be in one place, and it's already in two (one of which is a package.json devDependency).  Maybe it should be exposed in @pcd/gpc and/or @pcd/gpcircuits since that's the code which really needs to match up with this version.